src/api/app/models/review.rb

```diff
@@ -32,6 +32,35 @@ def reviewer
     self[:reviewer] || by_user || by_group || by_project || by_package
   end
 
+  def accepted_at
+    element = history_elements.last
+    if state == :accepted && element.type == 'HistoryElement::ReviewAccepted'
+
+      # Special case, return the acceptance time of the user when it got assigned from a group to a user
+      if /review assigend to user (?<user>.+)/ =~ element.comment
```

Comment:
ReviewAssigned is always from a group to a user IIRC. The only thing which can happen that it gets reverted and assigned to another user, so you would have multiple assignments from the group to (different?) users.